core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -41,6 +41,22 @@ package object config {
     .bytesConf(ByteUnit.MiB)
     .createWithDefaultString("1g")
 
+  private[spark] val EVENT_LOG_COMPRESS =
+    ConfigBuilder("spark.eventLog.compress").booleanConf.createWithDefault(false)
```

Comment:
I have a slight preference for multi-line configs even when they fit in one line (makes it easier to see the type and default value).

Reply:
Okay, I'll change all the event logging configs.